lib/utils.js

```diff
@@ -270,6 +270,8 @@ const getChangelogLabelFromPullRequest = (pullRequest) => {
   );
   if (label) {
     return label.name.trim();
+  } else {
+    return false;
```

Comment:
`false` doesn't make sense here, maybe `null`

Reply:
addressed in latest commit.